moduleroot/Dockerfile.erb

```
@@ -1,4 +1,4 @@
-FROM ruby:2.5.3
+FROM ruby:2.5.8
```

I wonder if it should just pull from `ruby:2.5` so we don't need to keep it updated manually:
```suggestion
FROM ruby:2.5
```

---

Yes, good idea!